src/app/pages/onboarding/set-password/components/password-field.utils.ts

```diff
@@ -1,6 +1,6 @@
 import { PasswordStrength, ValidatedPassword } from '@app/common/validation/validate-password';
 
-export const defaultColor = 'accent.text-subdued';
+export const defaultColor = 'accent.background-secondary';
```

Comment:
I changed this as it wasn't appearing with `'accent.text-subdued'`
![Screenshot 2023-11-15 at 11 48 55](https://github.com/leather-wallet/extension/assets/2938440/07f46471-883d-4a3c-adfb-4e4e8063c1ad)